docs/guidelines/repository-guidelines/01-new-repository-settings.md

```diff
@@ -91,9 +52,6 @@ If you define any governance-related [Prow job](https://github.com/kyma-project/
 
 ## Create labels
 
-[Define labels](https://help.github.com/en/articles/creating-a-label) for the new repository so you could use them in issues and pull requests. Follow the naming convention and color array used in other repositories such as [`kyma`](https://github.com/kyma-project/kyma/labels).
-
-> **TIP:** You can copy labels from the existing repository to the new repository using the [GitHub CLI tool](https://github.com/jvandemo/copy-github-labels-cli).
+Labels are managed by Prow in the [`labels.yaml`](https://github.com/kyma-project/test-infra/blob/main/prow/labels.yaml) configuration file. To add a new label, create a Pull Request to the [`test-infra`](https://github.com/kyma-project/test-infra) repository with a new label definition. When adding a new label, follow the structure of the `labels.yaml` file.
 
-## Add the repository to teams
-Add the repository to both [developers](https://github.com/orgs/kyma-project/teams/developers/repositories) and [developers-triage](https://github.com/orgs/kyma-project/teams/developers-triage/repositories) teams.
+Additionally, you can [define repository labels](https://help.github.com/en/articles/creating-a-label) for the new repository so you could use them in issues and pull requests. Follow the naming convention and color array used in other repositories such as [`kyma`](https://github.com/kyma-project/kyma/labels).
```

Comment:
As discussed: please update this paragraph and/or explain when we should create labels using PRs to the `test-infra` repo, and when using the method described in this article.